dev/ci/ci-fiat-parsers.sh

```diff
@@ -3,8 +3,6 @@
 ci_dir="$(dirname "$0")"
 . "${ci_dir}/ci-common.sh"
 
-fiat_parsers_CI_DIR="${CI_BUILD_DIR}/fiat"
+git_download fiat_parsers
```

Comment:
I believe you're missing `FORCE_GIT=1` here; the fiat-parsers Makefile tries to update / init submodules itself.

Reply:
Feel free to also update the submodule in the build script.  In any case, I believe I've fixed the silent incorrect success with https://github.com/mit-plv/fiat/commit/d595813799d2f141fcba9c8c66c5308783a4ccb2